flink-end-to-end-tests/test-scripts/test_mesos_multi_submission.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+################################################################################
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+set -o pipefail
+
+source "$(dirname "$0")"/common.sh
+source "$(dirname "$0")"/common_mesos_docker.sh
+
+FIRST_DURATION=15
+SECOND_DURATION=5
+CPUS=$(lscpu | grep "^CPU(s):" | awk '{print $2}')
```

Comment:
Why assign the value here? `CPUS` will get re-assigned anyways, or the script won't continue.

I suppose `CPUS` must match the auto-discovered value from the mesos agent. Do we actually have to determine the number of CPUs dynamically or can we use a constant that is independent from the hardware [1]? 

[1] http://mesos.apache.org/documentation/attributes-resources/#predefined-uses-conventions

Reply:
Good catch. It's a redundant line.
Yes, I have also considered that way. However, I've a concern makes me not choose that option. I think the auto-discovered value will free us from the the hardware limitation. For example, if we set it to 4, what if there is not enough cpu core in Travis or developer's environment? It's tricky to determine this value.

Reply:
I think there is no harm to start the Mesos agent with more CPU resources than the undlying hardware has to offer. The `cpus` resource value is virtual and should be only relevant for bookkeeping. Furthermore, it is not clear how Mesos determines the number of CPUs, and whether their automatically determined value will match ours. To keep things simple, we can just set the mesos CPU resources to `1`. Please let me know if I am missing something.

Reply:
Agreed to set it to 1 in this test case.